ui/app/rif/pages/domainsPage.js

```diff
@@ -18,6 +18,33 @@ const mockDomains = [
 		ownerAddress: '0x5F4df703Da966E12d3068E1aCbe930f2E363c732',
```

Comment:
Is this just mock data or the idea of domainsPage is to use it for save variable values? On that case, why dont use a static class or constants file?

Reply:
Mocks were moved to a folder, in case we want to use them to show functionalities